src/app/calendar/page.tsx

```diff
@@ -0,0 +1,18 @@
+import LandingPage from '@/components/big-calendar/calendar';
```

Comment:
Move this to /admin/calendar

Reply:
Kirat, in this on the calendar page admin has the option to add event, basically it shows a Add Event button on the calendar, so it is kept this way